pom.xml

```diff
@@ -42,6 +42,8 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<junit.platform.version>1.0.0-M3</junit.platform.version>
```

Comment:
Super !!! 

I like this. In future you will able to use maven versioning plugin to upgrade latest 3rd party dependencies